### PR TITLE
Add scripts to deploy kubevirt, cdi  and the old operator

### PR DESCRIPTION
**Pull request by akrejcir**
**What this PR does / why we need it**:
Added scripts used by openshift CI

```release-note
NONE
```

### CODE REVIEW

**Comment by akrejcir** [comment]
/cc @omeryahud, @vatsalparekh 
This PR is needed by: https://github.com/openshift/release/pull/13867

**Review by vatsalparekh — CHANGES_REQUESTED** [review]
Can you also add a make target?

**Comment by akrejcir** [comment]
These scripts are only helpers needed by the CI. I don't think they will be useful as makefile targets.

File: hack/deploy-kubevirt-and-cdi.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+EOF
+
+# Deploying kuebvirt
+LATEST_KUBEVIRT=$(curl -L https://api.github.com/repos/kubevirt/kubevirt/releases | \
+            jq '.[] | select(.prerelease==false) | .name' | sort -V | tail -n1 | tr -d '"')
+
+oc apply -n $NAMESPACE -f "https://github.com/kubevirt/kubevirt/releases/download/${LATEST_KUBEVIRT}/kubevirt-operator.yaml"
+oc apply -n $NAMESPACE -f "https://github.com/kubevirt/kubevirt/releases/download/${LATEST_KUBEVIRT}/kubevirt-cr.yaml"
+
+oc apply -n $NAMESPACE -f - <<EOF
```
Comment (jean-edouard) [inline hack/deploy-kubevirt-and-cdi.sh:20]: Nit: I believe the ConfigMap is the "legacy" way of configuring kubevirt. Feature gates are now configured in the CR, for example:
```
 spec:
  configuration:
    developerConfiguration:
      featureGates:
      - DataVolumes
      - CPUManager
      - LiveMigration
```

**Review by jean-edouard — APPROVED** [review]

File: hack/deploy-kubevirt-and-cdi.sh (hunk 1 above)
Comment (vatsalparekh) [reply hack/deploy-kubevirt-and-cdi.sh:20]: @akrejcir can you confirm this?
otherwise, /lgtm

Comment (ksimon1) [reply hack/deploy-kubevirt-and-cdi.sh:20]: It is like Jed saying https://pkg.go.dev/kubevirt.io/client-go/api/v1#DeveloperConfiguration

Comment (akrejcir) [reply hack/deploy-kubevirt-and-cdi.sh:20]: Changed to use `featureGates`.

**Review by jean-edouard — APPROVED** [review]

**Pull request merged**